massgov/mayflower
# patternlab DP-21660 list display after left float image in IE11

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Fix list bullets and numbers overwrap left floated elements in IE11.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-21660)
- [Github issue]()

## Steps to Test


1. Build the Drupal site with this Mayflower.
2. Add lists after a left floated element in the rich text editor.
3. See the page in IE11 and find bullets and numbers are not on the left apart from their content on the right.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
